cpp/contracts/adapter.cpp

```diff
@@ -189,8 +189,8 @@ void adapter::settle(const name& caller, const operation& operation, const metad
    adapter_registry_table registry_data = _registry.get();
    check(registry_data.token_bytes == operation.token, "underlying token does not match with adapter registry");
    // symbol check if underlying is EOS type
-   if (registry_data.token != name(0)) // TODO add test
-      check(registry_data.xerc20_symbol == operation.amount.symbol, "registered xerc20 symbols differs from the operation one");
+   // if (registry_data.token != name(0)) // TODO add test
+   //    check(registry_data.xerc20_symbol == operation.amount.symbol, "registered xerc20 symbols differs from the operation one");
```

Comment:
Do we still need this?

Reply:
No, symbol in the amount is gone indeed.